### PR TITLE
Update readme file.

### PR DESCRIPTION
**Pull request by e7506c**
Add some steps to readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,13 @@ Make sure you have installed all of the following prerequisites on your machine:
 ## Quick Start
 
 ```
+$ git clone https://github.com/nspcc-dev/neofs-dev-env.git
+$ make get
+$ make hosts
+```
+Then add ```make hosts``` command output to /etc/hosts file.
```
[the remark this answers is not in the document]
Comment (e7506c) [reply README.md]: Right. Thank you! Fixed.

**Review by alexvanin — APPROVED** [review]
Usually commit headers do not include period symbol. Besides that LGTM.

File: README.md [hunk 2]
```diff
@@ -23,8 +23,30 @@ Make sure you have installed all of the following prerequisites on your machine:
...
 ## Quick Start
+Clone repo: 
 
 ```
+$ git clone https://github.com/nspcc-dev/neofs-dev-env.git
+```
+
+Run next commands from project's root:
+```
+$ make get
```
Comment (realloc) [inline README.md:34]: `get` will be called on `up`. There is not much sense in doing it beforehand.

**Review by realloc — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (anatoly-bogatyrev) [reply README.md]: `make hosts` will not be executed without `make get`

**Pull request merged**